Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -361,10 +310,71 @@ protected override int CreateToolWindow(ref Guid toolWindowType, int id)
             config.WorkingDirectory = runTarget.WorkingDirectory;
 
-            RunProfiler(session, config, openReport);
+            if (useVTune) {
+                RunVTune(session, config, openReport);
+            } else {
+                RunProfiler(session, config, openReport);
+            }
         }
 
 
+        private static void RunVTune(SessionNode session, LaunchConfiguration config, bool openReport) {
+#if false
+            var pyexe = ProcessOutput.QuoteSingleArgument(stndTarget.InterpreterPath);
+
+            if (pyexe == string.Empty) {
+                if (stndTarget.PythonInterpreter != null) {
+                    var registry = session._serviceProvider.GetComponentModel().GetService<IInterpreterRegistryService>();
+                    var interpreter = registry.FindConfiguration(runTarget.PythonInterpreter.Id);
+                    if (interpreter == null) { 
+                        /* ??? */;
+                        MessageBox.Show("Could not find interpreter in the registry");
+                    } else {
+                        MessageBox.Show($"The Python interpreter in question is: [{interpreter}]");
+                    }
+                }
+            }
+#endif
+
+            var interpreter = config.GetInterpreterPath();
+            if (!File.Exists(interpreter)) {
+                MessageBox.Show("Can't find specified python interpreter.");
+                return;
+            }
+
+            string outPathDir = Path.GetTempPath();
+            string outPath = Path.Combine(outPathDir, "pythontrace.diagsession");
+
+            var driver = PythonToolsInstallPath.GetFile(ExternalProfilerDriverExe, typeof(PythonProfilingPackage).Assembly);
+
+            var procInfo = new ProcessStartInfo(driver) {
+                CreateNoWindow = false,
+                Arguments = string.Join(" ", new[] {
+                    "-d",
+                    ProcessOutput.QuoteSingleArgument(outPathDir),
+                    "--",
+                    ProcessOutput.QuoteSingleArgument(interpreter),
+                    config.InterpreterArguments,
+                    string.IsNullOrEmpty(config.ScriptName) ? "" : ProcessOutput.QuoteSingleArgument(config.ScriptName),
+                    config.ScriptArguments
+                }),
+                WorkingDirectory = config.WorkingDirectory,
+            };
+
+            var proc = new Process { StartInfo = procInfo };
+            var dte = (EnvDTE.DTE)session._serviceProvider.GetService(typeof(EnvDTE.DTE));
+            proc.EnableRaisingEvents = true;
+            proc.Exited += (_, args) => {
+                if (!File.Exists(Path.Combine(outPathDir, "Sample.dwjson"))) {
+                    MessageBox.Show($"Something happened, cannot find output file");
+                } else {
+                    PackageTrace(outPathDir);
+                    dte.ItemOperations.OpenFile(Path.Combine(outPathDir, "trace.diagsession"));
```

Comment:
If `PackageTrace` is the one creating this `trace.diagsession` file, then it would be better to have it return the path instead of hard coding the name in both places.